### PR TITLE
Add check for neutron ovn metadata issue

### PR DESCRIPTION
**Pull request by zhhuabj**

### CODE REVIEW

File: hotsos/defs/tests/scenarios/openstack/ovsdb_server_frequent_reelections.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+target-name: ovsdb_server_frequent_reelections.yaml
+data-root:
+  files:
+    var/log/neutron/neutron-ovn-metadata-agent.log: |
+      2022-08-18 12:30:36.103 75556 ERROR ovsdbapp.backend.ovs_idl.transaction [-] OVSDB Error: no error details available
```
Comment (dosaboy) [inline hotsos/defs/tests/scenarios/openstack/ovsdb_server_frequent_reelections.yaml:5]: nit: the tests data_root has:

```
$ cat tests/unit/fake_data_root/openstack/sos_commands/date/date
Thu Feb 10 16:19:17 UTC 2022
```

so while the timestamp you have added will work, it is in fact several months in the future of the date. this is just a nit but to bring it inline you could do:

```
2022-02-10 12:30:36.103 75556 ERROR ovsdbapp.backend.ovs_idl.transaction [-] OVSDB Error: no error details available
```

File: hotsos/defs/scenarios/openstack/ovsdb_server_frequent_reelections.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+conclusions:
+  lp1996594:
+    decision:
+      - has_1996594
+      - no_monitor_cond_since
+    raises:
+      type: LaunchpadBug
+      bug-id: 1996594
+      message: >-
+        Known metadata bug identified that metadata may randomly stops working.
```
Comment (dosaboy) [inline hotsos/defs/scenarios/openstack/ovsdb_server_frequent_reelections.yaml:26]: if i an propose a reword here:

neutron-ovn-metadata-agent is impacted by a known bug that can cause the agent to randomly stop processing metadata requests. Upgrading to a version of openvswitch >= 2.17.0 should resolve the problem.

Comment (zhhuabj) [reply hotsos/defs/scenarios/openstack/ovsdb_server_frequent_reelections.yaml:26]: This piece of English smells really fragrant

File: hotsos/defs/tests/scenarios/openstack/ovsdb_server_frequent_reelections.yaml (hunk 1 above)
Comment (zhhuabj) [reply hotsos/defs/tests/scenarios/openstack/ovsdb_server_frequent_reelections.yaml:5]: has changed to 2022-02-10

File: hotsos/defs/scenarios/openstack/ovsdb_server_frequent_reelections.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  no_monitor_cond_since:
+    apt:
+      openvswitch-common:
+        - min: 0
+          max: 2.17.0
+  has_1996594:
+    input:
+      path: 'var/log/neutron/neutron-ovn-metadata-agent.log'
+    search:
+      expr: '([0-9-]+)\s[0-9:\.]+.+OVSDB Error: no error details.+'
```
[the remark this answers is not in the document]
Comment (zhhuabj) [reply hotsos/defs/scenarios/openstack/ovsdb_server_frequent_reelections.yaml:11]: done

**Comment by dosaboy** [comment]
sorry i should have mentioned before but can you please do the following:

```
git mv hotsos/defs/scenarios/openstack/ovsdb_server_frequent_reelections.yaml hotsos/defs/tests/scenarios/openstack/neutron/bugs/lp1996594.yaml
git mv hotsos/defs/tests/scenarios/openstack/ovsdb_server_frequent_reelections.yaml hotsos/defs/tests/scenarios/openstack/neutron/bugs/lp1996594.yaml
```

and you can actually remove "target-name: ..." from the test as it shares the same filename as the target already

**Comment by dosaboy** [comment]
basically bug checks should always going into the respective bugs/ dir

**Comment by zhhuabj** [comment]
> 

Done, this is unit test result - https://pastebin.canonical.com/p/ywbV7hcmnX/

**Pull request merged**